bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/component.tsx

```
@@ -246,68 +263,58 @@
-                ref={sentinelRefProxy}
-                style={{
-                  height: 1,
-                  background: 'none',
-                }}
-              />
-            </MessageList>
-          </MessageListWrapper>,
+            />
+          </MessageList>,
           renderUnreadNotification,
+          <ChatReplyIntention />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)